blocks/newslist/newslist.css

```diff
@@ -48,17 +48,17 @@
 .newslist .newslist-container .newslist-item .newslist-item-footer a {
   font-family: var(--graphik-semibold);
   text-transform: uppercase;
-  font-size: 16px;
+  font-size: var(--body-font-size-xxs);
   font-weight: 600;
-  color: #004dff;
+  color: var(--link-color);
 }
 
 .newslist .newslist-container .newslist-item .newslist-item-footer a .read-more-arrow {
   display: inline-block;
   height: 20px;
   width: 40px;
   line-height: 20px;
-  background: transparent url('/icons/arrow-icon.svg') no-repeat 0 -8px;
+  background: 'transparent url(/icons/arrow-icon.svg) no-repeat 0px -8px';
```

Comment:
this is invalid css, the original one `transparent url('/icons/arrow-icon.svg') no-repeat 0 -8px` is correct